README.md

```diff
@@ -237,22 +237,22 @@ conda install -c conda-forge numpyro
    - Provide the `rng_key` argument to `numpyro.sample`. e.g. `numpyro.sample('x', dist.Normal(0, 1), rng_key=PRNGKey(0))`.
    - Wrap the code in a `seed` handler, used either as a context manager or as a function that wraps over the original callable. e.g.
 
-     ```python
-     with handlers.seed(rng_seed=0):  # random.PRNGKey(0) is used
-         x = numpyro.sample('x', dist.Beta(1, 1))    # uses a PRNGKey split from random.PRNGKey(0)
-         y = numpyro.sample('y', dist.Bernoulli(x))  # uses different PRNGKey split from the last one
-     ```
+        ```python
+        with handlers.seed(rng_seed=0):  # random.PRNGKey(0) is used
+            x = numpyro.sample('x', dist.Beta(1, 1))    # uses a PRNGKey split from random.PRNGKey(0)
+            y = numpyro.sample('y', dist.Bernoulli(x))  # uses different PRNGKey split from the last one
+        ```
```

Comment:
This indentation is an attempt to make python code rendered correctly [here](http://num.pyro.ai/en/stable/getting_started.html#frequently-asked-questions). This only happens on readthedocs, I can't reproduce in my system.